flepimop/R_packages/inference/inst/scripts/flepimop-inference-main.R

```diff
@@ -124,10 +124,12 @@ foreach(seir_modifiers_scenario = seir_modifiers_scenarios) %:%
     if (nchar(opt$ground_truth_end) > 0) {
       ground_truth_end_text <- c("--ground_truth_end", opt$ground_truth_end)
     }
-    
-    err <- system(
-      paste(
-        "flepimop-inference-slot",
+
+    log_file <- paste0(
+        "log_inference_slot_", config$name, "_", opt$run_id, "_", flepi_slot, ".txt"
+    )
+    command <- c(
+        file.path(opt$flepi_path, "flepimop", "main_scripts","inference_slot.R"),
```

Comment:
this has changed w/ the refactor. Should now be something like `flepimop-inference-slot` as its on the path as a runnable script?

not exactly sure how that plays with windows, however.

Reply:
Good catch, addressed in f0a70d27d9055e04cd42a060713a12831b9e37d7.